### PR TITLE
Component Upgrades

### PR DESCRIPTION
**Pull request by darranl**
https://issues.redhat.com/browse/ELYEE-41
https://issues.redhat.com/browse/ELYEE-42
https://issues.redhat.com/browse/ELYEE-43

### CODE REVIEW

**Review by fjuma — APPROVED** [review]

**Review by skyllarr — APPROVED** [review]

**Comment by skyllarr** [comment]
thanks @darranl  , I've approved this, but I don't have write access to this repo so I can't merge this

**Pull request merged**
